http/transport.go

```diff
@@ -0,0 +1,48 @@
+// Shared transports for all clients to prevent leaking connections
+package http
+
+import (
+	"crypto/tls"
+	"net"
+	"net/http"
+	"time"
+)
+
+// defaultTransport is the default implementation of Transport and is
```

Comment:
Would you mention that this is the same as http.DefaultTransport (and if there are values that are different than in the standard library, please note 'em)